recipes/cmec_driver/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "cmec_driver" %}
+{% set version = "1.0.0a5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/cmecmetrics/cmec-driver/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 5620acc7787b155c4f7c3e5922681e958528f23dd41170a28c7dbb90771fe184
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+entry_points:
+  -cmec-driver = cmec_driver:main
```

Comment:
The `entry_points` must be in the build section. That should fix the Windows build BTW.

Reply:
Thanks @ocefpaf!